external/stdk/README.md

```diff
@@ -0,0 +1,48 @@
+# SmartThings Device SDK
+
+The SmartThings Device SDK(for short STDK) is the core device library that allow device applications to securely connect to the SmartThings Cloud. 
+It has been integrated on TizenRT as a submodule. 
+
+## Prerequisites
+
+- Mobile phone with SmartThings application installed.
+- Wifi AP could connect internet.
+- An Artik053 board or ESP32 Wrover kit board. (with necessary devices connected)
+  Artik053, input button use SW703 on board, RGB LED connect GPIO 53(R), 51(G), 52(B), and GND connect 3.3v
+  ESP32 Wrover, input button connect GPIO 5, RGB LED connect GPIO 18(R), 21(G), 19(B), and GND connect 3.3v
+
+## How to build with STDK
+
+### Update submodule
```

Comment:
```suggestion
1. Update submodule
```
How about using serial number here instead of ### ?

How to ...? step...
1. Update submodule
2. Apply patch for stdk
3. Register device and update configure info
4. Select build config

Reply:
under the md view, the font is smaller for these steps, seems enough to distinguish them. let's just keep it.

Reply:
You can write in this way:
```text
### 1. Update submodule
### 2. Apply patch for stdk
### 3. Register device and update configure info
### 4. Select build config
```
then, you can get: 
### 1. Update submodule
### 2. Apply patch for stdk
### 3. Register device and update configure info
### 4. Select build config